src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1528,7 +1531,7 @@ private void CheckMemberNameConflicts(DiagnosticBag diagnostics)
 
                     var conversion = symbol as SourceUserDefinedConversionSymbol;
                     var method = symbol as SourceMemberMethodSymbol;
-                    if ((object)conversion != null)
+                    if (!(conversion is null))
```

Comment:
>!(conversion is null) [](start = 24, length = 21)

Consider `is object` instead.

Reply:
Never!

Reply:
When you say never, you're being inconsistent with the rest of the compiler codebase.

Reply:
I am not convinced by the assertion that `is object` is a clear and straightforward way to test for null.

Reply:
I am not convinced by the assertion that `is object` is a clear and straightforward way to test for null.

Reply:
There are plenty of places we don't use `is object` in the code base today. Until we settle on a single pattern, any of them are acceptable.